futurenhs.app/components/_pageTemplates/SiteUserTemplate/index.tsx

```diff
@@ -1,50 +1,141 @@
+    const shouldRenderEditButton: boolean = hasEditPermissions && !isEditMode;
+
+    const siteUserInitials: string = initials({ value: `${siteUser.firstName} ${siteUser.lastName}` });
+
+
+    /**
+     * Client-side submission handler
+     */
+    const handleSubmit = async (formData: FormData): Promise<FormErrors> => {
+
+        try {
+
+            //TODO: PENDING API
+            
+
+        } catch (error) {
+
+            const errors: FormErrors = getServiceErrorDataValidationErrors(error) || getGenericFormError(error);
+
+            setErrors(errors);
+
+            return Promise.resolve(errors);
+
+        }
+
+    };
+
+
+    /**
+     * Render
+     */
     return (
-        <LayoutColumn className="c-page-body">
-            <BackLink
-                href={routes.siteRoot}
-                text={{
-                    link: 'Back',
-                }}
-            />
-            <LayoutColumnContainer justify="centre">
-                <LayoutColumn tablet={11}>
-                    <UserProfile
-                        headingLevel={1}
-                        profile={siteUser}
-                        text={{
-                            heading: secondaryHeading,
-                            firstNameLabel: firstNameLabel,
-                            lastNameLabel: lastNameLabel,
-                            pronounsLabel: pronounsLabel,
-                            emailLabel: emailLabel,
-                        }}
-                        className="tablet:u-justify-center tablet:u-mt-16"
-                    />
-                </LayoutColumn>
-            </LayoutColumnContainer>
-        </LayoutColumn>
+            <LayoutColumn className="c-page-body">
+                <LayoutColumnContainer>
+                    <LayoutColumn tablet={6}>
+                        {isEditMode ?
+
+                            hasEditPermissions &&
+                            <>
+                                <h1>{editHeading}</h1>
+                                <Avatar
+                                    image={siteUser.image}
+                                    initials={siteUserInitials}
+                                    className="u-h-36 u-w-36 u-mb-8"
+                                />
+                                <FormWithErrorSummary
+                                    csrfToken={csrfToken}
+                                    formConfig={formConfig}
+                                    errors={errors}
+                                    text={{
+                                        form: {
+                                            submitButton: 'Save changes',
+                                            cancelButton: 'Discard changes'
+                                        }
+                                    }}
+                                    submitAction={handleSubmit}
+                                    cancelHref={`/users/${siteUser.id}`}
```

Comment:
can we add /users to the withRoutes hof so we can access that via props.routes and not need to hardcode here?

Reply:
Yep now added this